### DISCUSSION TITLE
【灌水】大家连绳网用的虚空终端还是联觉信标啊？

### DISCUSSION THREAD

**Discussion by FRIKY-c** [share121/inter-knot#2904] [其他]
最近感觉**_H.D.D_**有点卡了，想问问各位有没有新设备推荐

**Comment by LuAn-afl** [comment]
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >我不知道为啥我看不见图片

**Reply by Nie7bai** [reply]
可能是加载问题，还没加载完，可以挂个加速器在后台

**Comment by haxyyx** [comment]
用Windows 可以无障碍远程邦布
